src/utils/index.ts

```diff
@@ -8,14 +9,41 @@ const hasError = (response: any) => {
     return !!response.data._ERROR_MESSAGE_ || !!response.data._ERROR_MESSAGE_LIST_;
 }
 
-const showToast = async (message: string) => {
+const showToast = async (message: string, err?: any) => {
+  if(message === "Something went wrong" && err) {
     const toast = await toastController
-        .create({
-          message,
-          duration: 3000,
-          position: 'bottom',
-        })
-      return toast.present();
+    .create({
+      message,
```

Comment:
I thin we could create a initialisation object 
```
{
      message,
      duration: 3000,
      position: 'bottom'
}
```

Add buttons to it if the err is passed